lib/graphql/client.ts

```diff
@@ -1,19 +1,42 @@
 import { GraphQLClient as GQLClient } from "graphql-request";
+import { SubscriptionClient } from "subscriptions-transport-ws";
 import { Variables } from "graphql-request/dist/src/types";
+import * as ws from "ws";
 
 import { Auth } from "../auth";
-import { RawQueryResponse } from "./types";
+import { RawQueryResponse, SubscriptionType, Unsubscribe } from "./types";
 import { serializeGraphQLError } from "../utils";
 import { GraphQLError, UserUnauthorizedError } from "../errors";
+import { GraphQLClientOpts } from "../types";
+
+type Subscriptions = {
```

Comment:
these types are kinda local so I thought it was fine to keep them in this file, but I can definitely extract them if it looks more proper

Reply:
Agree.